### ISSUE TITLE
Add a conforming TypeScript @types/imagemin-webp-webpack-plugin definition

### ISSUE THREAD

**Issue by Diono** [iampava/imagemin-webp-webpack-plugin#58]
It is now possible to work your Webpack project in Typescript. One of the main interests is to better follow the evolutions of the dependent modules, the obsolete options or methods... It would be good to add a project @types/imagemin-webp-webpack-plugin :)

**Comment by iampava** [comment]
I agree, it would be nice to have the plugin written in TypeScript :)

Would you be interested in doing a PR with this?

**Comment by Diono** [comment]
Unfortunately I do not yet have the skills and experience of webpack modules (hooks?). I still have a beginner level in typescript, I know how to use it on my projects compiled with webpack ... but it ends there :(

**Comment by iampava** [comment]
Got it. Will try and find some time to handle this :)
